tests/e2e/go.mod

```diff
@@ -8,6 +8,54 @@ require (
 	github.com/octago/sflags v0.2.0
 	github.com/spf13/pflag v1.0.5
 	gopkg.in/yaml.v2 v2.3.0
+	k8s.io/apimachinery v0.20.0
 	k8s.io/klog/v2 v2.4.0
+	k8s.io/kops v0.0.0-00010101000000-000000000000
 	sigs.k8s.io/kubetest2 v0.0.0-20210119223558-43699a7ba223
 )
+
+replace k8s.io/kops => ../../.
+
+// These should match the go.mod from k8s.io/kops
+
+replace k8s.io/api => k8s.io/api v0.20.0
```

Comment:
I'm wondering if we should replace these the relative path to the local vendor directory, that way we dont need to keep two go.mod files in sync anytime we bump these.

Reply:
Just tried this locally and it doesn't work. `replace k8s.io/api => ../../vendor/k8s.io/api` will fail because go expects the go.mod and go.sum files to exist at the location, but vendoring modules explicitly does not include them.

Looks like we'll just need to keep these in sync going forward